disk/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG - disk
 
+## 4.1.0 / 2021-01-25
+
+* [Added] Support network drives on Windows. See [#8273](https://github.com/DataDog/integrations-core/pull/8273).
+* [Added] Add device_label tag in addition to label tag.. See [#8306](https://github.com/DataDog/integrations-core/pull/8306).
```

Comment:
```suggestion
* [Added] Add device_label tag in addition to label tag. See [#8306](https://github.com/DataDog/integrations-core/pull/8306).
```

Maybe the release tool should not add a period at the end of a PR title if there is one already.

Reply:
Yes I thought the same, I filed https://github.com/DataDog/integrations-core/pull/8442 :+1: